src/common/NavigationConstants.js

```diff
@@ -1,4 +1,7 @@
-// constants that define navigation routes
+/**
+ * Constants that define navigation routes.
+ * Warning: server/index.js expects each value to be a string.
```

Comment:
Initially I tried adding the `VIEW: { METADATA: "metadata" }` object to these default constants, but I got a `string.replace is not a function` error message (without a stack trace) that I eventually tracked down to `server/index.js`.

So I added this comment to prevent others from running into the same issue 😅